### DISCUSSION TITLE
General Conversations

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#53] [General]
Lets Please keep it clean

**Comment by titof2375** [comment]
hello, it's influxd 1 or version 2

**Reply by tteck** [reply]
Influxdb-Version: 1.8.10

**Comment by titof2375** [comment]
hello how can you modify the mp root?

**Comment by titof2375** [comment]
hello how to remove the automatic authentication, and authorize the connection to putty in root?

**Reply by tteck** [reply]
to obtain a root password, type `passwd root` in the LXC console and set one

**Comment by titof2375** [comment]
and for this connect with putty as root?

**Comment by pchouard** [comment]
Hi there, sorry for this most likely stupid question, but I cannot login into the MQTT CT console under Proxmox. I tried root and my default password and it doesn't work. 
I also had a look into the scripts and didn't see anything. What is the default password please?
By the way, many thanks for your fantastic job!

**Reply by tteck** [reply]
If it's asking for a password, the script didn't finish
remove CT and run the script again. Wait for "Successfully created MQTT LXC"

**Reply by pchouard** [reply]
Thanks for your prompt reply.
CT is successfully installed and running but when I get the prompt "mqtt login:" in the CT console I have no idea of the credentials to use.
I of course tried those of my Proxmox install.
I had no prompt during the installation process to type some.

**Reply by tteck** [reply]
again, If it's asking for a credentials, the script didn't finish. It will never ask for login or password if the script finished successfully

**Reply by pchouard** [reply]
copy that. I'll try again. Thx.
